VisualPinball.Unity/VisualPinball.Unity/Extensions/MaterialExtensions.cs

```diff
@@ -16,15 +16,9 @@ public static class MaterialExtensions
 		private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
 
 		/// <summary>
-		/// Lazy instantiation of the material converter
+		/// Material Converter instance for the current graphics pipeline
 		/// </summary>
-		private static readonly Lazy<IMaterialConverter> lazyMaterialConverterInstance = new Lazy<IMaterialConverter>(() => CreateMaterialConverter());
-
-		/// <summary>
-		/// Singleton Material Converter
-		/// </summary>
-		public static IMaterialConverter MaterialConverter { get { return lazyMaterialConverterInstance.Value; } }
-
+		public static IMaterialConverter MaterialConverter => CreateMaterialConverter();

```

Comment:
Sorry didn't mean the idea of a singleton was bad, was just curious about the benefits of Lazy over a simple singleton one liner. I think we probably would prefer not to instantiate these very call

Reply:
Reading [this](https://docs.microsoft.com/en-us/dotnet/framework/performance/lazy-initialization) it simply seems a thread-safe way of doing the same thing. If it's part of .NET Standard and supported by Unity I don't mind using it.